supla-common/proto.c

```diff
@@ -109,6 +113,7 @@ unsigned char PROTO_ICACHE_FLASH sproto_buffer_append(
     void *spd_ptr, char **buffer, unsigned _supla_int_t *buffer_size,
     unsigned _supla_int_t *buffer_data_size, char *data,
     unsigned _supla_int_t data_size) {
+  (void)(spd_ptr);
```

Comment:
What is the purpose of this?

Reply:
It removes compilator warning about not used parameter.